README.md

```diff
@@ -117,22 +134,8 @@ Finally, enable all of the rules that you would like to use.
 
 If you're searching for React Native specific linting rules, check out [eslint-plugin-react-native](https://github.com/Intellicode/eslint-plugin-react-native).
 
-# Recommended configuration
-
-This plugin export a `recommended` configuration that enforce React good practices.
 
-To enable this configuration use the `extends` property in your `.eslintrc` config file:
 
-```js
-{
-  "plugins": [
-    "react"
-  ],
-  "extends": "plugin:react/recommended"
-}
-```
-
-See [ESLint documentation](http://eslint.org/docs/user-guide/configuring#extending-configuration-files) for more information about extending configuration files.
 
 The rules enabled in this configuration are:
 
```

Comment:
The enabled rule list part should also be moved. 

But, while I think it's a good idea to make the recommended configuration more visible, it will be a bit weird to have this rule list before the actual full rule list.